src/reynir_correct/checker.py

```diff
@@ -395,29 +401,31 @@ def create_sentence(self, job: Job, s: TokenList) -> Sentence:
         return sent
 
 
-def check_single(sentence_text: str, **options: Any) -> Optional[Sentence]:
+def check_single(
+    sentence_text: str, rc: GreynirCorrect, **options: Any
+) -> Optional[Sentence]:
     """Check and annotate a single sentence, given in plain text"""
     # Returns None if no sentence was parsed
     max_sent_tokens = options.pop("max_sent_tokens", DEFAULT_MAX_SENT_TOKENS)
-    rc = GreynirCorrect(**options)
     return rc.parse_single(sentence_text, max_sent_tokens=max_sent_tokens)
 
 
-def check_tokens(tokens: Iterable[CorrectToken], **options: Any) -> Optional[Sentence]:
+def check_tokens(
+    tokens: Iterable[CorrectToken], rc: GreynirCorrect, **options: Any
+) -> Optional[Sentence]:
```

Comment:
Same comment here as for check_single().

Reply:
Fixed